java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -64,30 +64,30 @@ public void defaultConstructor_instantiated_setsConcreteDriverManager()
     @Test
     public void startSession_setsBrowserKey() {
         String expectedBrowserCapabilityKey = "browserName";
-        String actualBrowser = sauce.sauceSessionCapabilities.getCapability(expectedBrowserCapabilityKey).toString();
+        String actualBrowser = sauce.currentSessionCapabilities.getCapability(expectedBrowserCapabilityKey).toString();
         assertNotEquals("", actualBrowser);
     }
     @Test
     public void start_setsPlatformNameKey() {
         String correctPlatformKey = "platformName";
-        String browserSetInSauceSession = sauce.sauceSessionCapabilities.getCapability(correctPlatformKey).toString();
+        String browserSetInSauceSession = sauce.currentSessionCapabilities.getCapability(correctPlatformKey).toString();
```

Comment:
@joshmgrant what do you think about the name change to `currentSessionCapabilities`? I think it makes sense for what it does. But I wanted your thoughts as well

Reply:
yeah this is a good name I think @nikolay-advolodkin